### DISCUSSION TITLE
How to add a table in an Outline document?

### DISCUSSION THREAD

**Discussion by Lc0rE** [outline/outline#2242] [Self Hosting] [answered]
I'm trying to add a simple table in my Outline document, without any results.
Is it possible to add tables in an Outline document at all?

**Comment by tommoor** [comment, marked as answer]
On an empty line hit the "+" icon on the left and look for "Table". You can also just type `/table` !

**Reply by tommoor** [reply]
Update: It became apparent today that the table option disappeared for some hours so this is likely why you couldn't find it previously. Sorry about that, the bug has been fixed.
